extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/render/SwitchRenderer.java

```diff
@@ -82,7 +82,8 @@ public boolean canRender(Widget w) {
                 String button = getSnippet("button");
                 button = StringUtils.replace(button, "%item%", w.getItem());
                 button = StringUtils.replace(button, "%cmd%", mapping.getCmd());
```

Comment:
Also, using `StringEscapeUtils.escapeHTML` here is probably not the right choice — only quotes and backslashes need to be prepended with `\` so resulting HTML attribute has correct value when read from Javascript.